substrate/frame/assets/src/functions.rs

```diff
@@ -491,7 +491,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		let d = Asset::<T, I>::get(&id).ok_or(Error::<T, I>::Unknown)?;
 		ensure!(
 			d.status == AssetStatus::Live || d.status == AssetStatus::Frozen,
-			Error::<T, I>::AssetNotLive
+			Error::<T, I>::IncorrectStatus
```

Comment:
Can't we say `AssetNotLiveOrFrozen`? Not sure if `IncorrectStatus` is much helpful.

Reply:
I'm personally fine with `IncorrectStatus`. It's non-ambiguous since it's only used in one place, and allows for us to add new statuses in the future without needing to update the error.

Reply:
`AssetNotLiveNorFrozen` or `AssetNotLiveOrFrozen` ?
I will do.

Reply:
>  allows for us to add new statuses in the future without needing to update the error.

Yeah, it helps us, but not sure someone using a UI would get any info with the `IncorrectStatus` error. Either ways, its fine but I would prefer the errors to be more meaningful for the users.

Reply:
I don't think Rust enum errors are supposed to be shown directly to end-users in UIs? I always understood them as more for developers.

Reply:
Some UI might show the doc: `/// The asset status is not the expected status.`

Considering there is 3 status I think it is ok